specification/cognitiveservices/data-plane/ContentModerator/v1.0/contentmoderatorListManagement.swagger.json

```diff
@@ -0,0 +1,1446 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Content Moderator List Management Client",
+    "version": "1.0",
+    "description": "Use this API to create and manage your custom lists of Images and Text. The lists that you create can be used in the Image/Match and Text/Screen APIs."
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{azureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "$ref": "#/parameters/AzureRegion"
+      }
+    ]
+  },
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/contentmoderator/lists/v1.0/imagelists/{listId}": {
+      "get": {
+        "tags": [
+          "ImageList"
+        ],
+        "operationId": "ImageList_GetListIdDetails",
```

Comment:
given that `listId` is a parameter, I'd be ok with calling this `ImageList_GetDetails` -- specifying the middle seems redundant